lib/cqm/converter/utils.rb

```diff
@@ -7,7 +7,8 @@ def self.qdm_codes_to_hds_codes(qdm_codes)
       codes = {}
       qdm_codes.each do |qdm_code|
         qdm_code = qdm_code.stringify_keys
-        code_system = qdm_code['code_system']
+        code_system = qdm_code['codeSystem']
+        code_system = qdm_code['code_system'] if qdm_code['code_system']
```

Comment:
Are there instances where either of `codeSystem` or `code_system` can be set? I was under the impression that it was only one of the two.

Reply:
There are, this catches both cases.